src/components/pages/UserPage.js

```diff
@@ -1,13 +1,14 @@
 import React, { useEffect, useState } from 'react';
-import { Link } from 'react-router-dom';
+import { Link, Redirect } from 'react-router-dom';
 import D3Chart from '../chart/D3Chart';
 import API from '../../utils/api';
+import PropTypes from 'prop-types';
 
-export const UserPage = () => {
+export const UserPage = (props) => {
   const [repos, setRepos] = useState([]);
   const [d3Data, setD3Data] = useState({});
-  const [username, setUsername] = useState('davidholyko');
-  // TODO set username to props.match.params;
+  const [username, setUsername] = useState(props.match.params.user);
```

Comment:
now that I think about it, username shouldnt actually be in state. but we still need some kind of state for the input field